test/test_sender.py

```diff
@@ -103,6 +104,7 @@ def init_queue_with_item(queue, item=None):
         queue.put(item)
 
 
+@pytest.mark.skip('re-enable when we rewrite these tests')
```

Comment:
Can we redo the tests before merging the RB? Otherwise I don't think they'll ever get rewritten...

Reply:
Fixed this